src/controllers/users.controller.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable require-jsdoc */
+/* eslint-disable valid-jsdoc */
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import validateUser from '../helpers';
+import { users } from '../database/models';
+
+dotenv.config();
+class UserController {
+  /**
+    * User can be able to sign up
+    * @param {object} req
+    * @param {object} res
+    */
+  static async signUp(req, res) {
+    try {
+      const { error } = validateUser.validateSignup(req.body);
+      if (error) {
+        throw new Error(error.details[0].message);
+      }
+      const hashedPassword = await bcrypt.hash(req.body.password, 10);
```

Comment:
@Willy I think this should be in a helper function since it is reusable code

Reply:
The feedback implemented